builder/openstack/run_config.go

```diff
@@ -66,8 +67,8 @@ func (c *RunConfig) Prepare(ctx *interpolate.Context) []error {
 		}
 	}
 
-	if c.SourceImage == "" && c.SourceImageName == "" {
-		errs = append(errs, errors.New("Either a source_image or a source_image_name must be specified"))
+	if c.SourceImage == "" && c.SourceImageName == "" && c.SourceImageFilters.Filters == nil {
+		errs = append(errs, errors.New("Either a source_image, a source_image_name, or  must be specified"))
```

Comment:
Missing some words here.

Reply:
Do you mean MostRecent? I suppose that's a potential case. But I wouldn't expect users to base their Packer configuration on whatever the latest of all images visibile to them are. I'll fix this if you would like it to check that too though.

Reply:
OH. 

It's even more sad I missed it the second time around. I'm ashamed. Fixed.